src/Symfony/Bundle/FrameworkBundle/Test/WebTestAssertions.php

```diff
@@ -0,0 +1,413 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\FrameworkBundle\Test;
+
+use PHPUnit\Framework\Assert;
+use Symfony\Bundle\FrameworkBundle\Client;
+use Symfony\Component\HttpFoundation\Cookie;
+
+/**
+ * Ideas borrowed from Laravel Dusk's assertions.
+ *
+ * @see https://laravel.com/docs/5.7/dusk#available-assertions
+ */
+trait WebTestAssertions
+{
+    /** @var Client|null */
+    protected static $client;
+
+    public static function assertResponseIsSuccessful(): void
+    {
+        static::checkResponseAvailable();
+
+        $response = static::$client->getResponse();
+
+        Assert::assertTrue(
+            $response->isSuccessful(),
+            "Response was expected to be successful, but actual HTTP code is {$response->getStatusCode()}."
+        );
+    }
+
+    public static function assertHttpCodeEquals(int $expectedCode): void
+    {
+        static::checkResponseAvailable();
+
+        $response = static::$client->getResponse();
+
+        Assert::assertSame(
+            $expectedCode,
+            $code = $response->getStatusCode(),
+            "Response code [$expectedCode] does not match actual HTTP code [$code]."
+        );
+    }
+
+    public static function assertResponseHasHeader(string $headerName): void
+    {
+        static::checkResponseAvailable();
+
+        $response = static::$client->getResponse();
+
+        Assert::assertTrue(
+            $response->headers->has($headerName),
+            "Header [$headerName] was not found in the Response."
+        );
+    }
+
+    public static function assertResponseNotHasHeader(string $headerName): void
+    {
+        static::checkResponseAvailable();
+
+        $response = static::$client->getResponse();
+
+        Assert::assertFalse(
+            $response->headers->has($headerName),
+            "Header [$headerName] was not expected to be found in the Response."
+        );
+    }
+
+    public static function assertResponseHeaderEquals(string $headerName, string $expectedValue): void
+    {
+        static::checkResponseAvailable();
+
+        $response = static::$client->getResponse();
+
+        Assert::assertSame(
+            $expectedValue,
+            $value = $response->headers->get($headerName),
```

Comment:
i think we should test all header values here, i.e. `get($headerName, null, true)`

Reply:
Thanks, solved!